src/pymor/analyticalproblems/functions.py

```diff
@@ -401,43 +402,97 @@ def evaluate(self, x, mu=None):
 
 
 class BitmapFunction(Function):
-    """Define a 2D |Function| via a grayscale image.
+    """Define a piecewise constant 2D |Function| via a bitmap array.
 
     Parameters
     ----------
-    filename
-        Path of the image representing the function.
+    bitmap
+        2d |NumPy array| of the discrete function values.
     bounding_box
         Lower left and upper right coordinates of the domain of the function.
-    range
-        A pixel of value p is mapped to `(p / 255.) * range[1] + range[0]`.
```

Comment:
Maybe remove this after deprecation, i.e. after the release?

Reply:
I have re-added some documentation.